samples/templates/default-azuredeploy-docker.json

```diff
@@ -530,7 +578,7 @@
                 "template": {
                     "$schema": "https://schema.management.azure.com/schemas/2019-04-01/deploymentTemplate.json#",
                     "contentVersion": "1.0.0.0",
-                   
+
```

Comment:
NIT: empty line

Reply:
@abiisnn I had made the changes to remove the additional blank line. Changes are committed into this PR.